Jakefile.js

```diff
@@ -976,6 +976,7 @@ var servicesLintTargets = [
 var lintTargets = compilerSources
     .concat(harnessCoreSources)
     .concat(serverCoreSources)
+    .concat(["client.ts"].map(function(f) { return path.join(serverDirectory, f); }))

```

Comment:
seems like `client.ts` should just be added to `serverCoreSources`. I don't know the server code at all; is there some reason for keeping it separate?